tools/setup_git_secrets.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
I think I agree with you, although theoretically we could add something to the hooks to run the script adding all our secrets before making a commit.